src/lib/cache/gdfs_cache.hpp

```diff
@@ -130,9 +157,9 @@ class GDFSCache : public AbstractCacheImpl<Key, Value> {
     return (*it->second).priority;
   }
 
-  ErasedIterator begin() { return ErasedIterator{std::make_unique<Iterator>(_map.begin())}; }
+  ErasedIterator unsafe_begin() { return ErasedIterator{std::make_unique<Iterator>(_map.begin())}; }
```

Comment:
Now that we have `snapshot()`, do you still need this? @Bensk1 @Bouncner

Reply:
Too long ago this discussion. But the purpose of `snapshot()` was to avoid the potential problems with the iterators, right?

If so (and I don't oversee something), I am happy with using `snapshopt()` and removing the iterators.

Reply:
Did we only use these unsafe iterators for our plugins? In that case, that would be fine for me.